src/tests/App.test.js

```diff
@@ -1,5 +1,5 @@
 import { render, screen } from '@testing-library/react';
-import App from './App';
+import App from "../containers/App"
```

Comment:
Hello @Croustys 👋
Thank you for catching this. 🙌  We're almost there!

Can you kindly wrap the path in single quotes `' '` instead of double quotes and terminate the statement with a semicolon `;`

<img width="805" alt="Screen Shot 2022-10-05 at 9 47 11 AM" src="https://user-images.githubusercontent.com/81635839/194092010-bba1c044-da2a-4df4-bac7-a54e8fa725c1.png">


Once those changes have been made your PR will be approved!

Reply:
Sure thing!